### DISCUSSION TITLE
Cotas para los valores del head en la parte 2.

### DISCUSSION THREAD

**Discussion by antoniablanco** [IIC2133-PUC/2024-1#84] [Tarea 1] [answered]
Hola, se sabe cuales son las cotas para los valores entregados (id, year, x, y, region)? Por ejemplo ¿los años no superaran 10^5?, o las ¿regiones superan 10^3?.

**Comment by sachondo7** [comment, marked as answer]
Ninguna supera las cotas de un int, es decir I16
[acá más info](https://labviewwiki.org/wiki/Integer_data_type)
